app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt

```diff
@@ -221,11 +228,9 @@ class ProfileProgressFragmentTest {
       )
     }
   }
-  /* ktlint-enable max-line-length */
 
-  /* ktlint-disable max-line-length */
   @Test
-  fun testProfileProgressFragmentWithProgress_recyclerViewItem0_checkOngoingTopicsCount_countIsTwo() {
+  fun testProfileProgressFragmentWithProgress_recyclerViewItem0_checkOngoingTopicsCount_countIsTwo() {  // ktlint-disable max-line-length */
```

Comment:
` Unnecessary space(s)` multiple spaces before comment and after `{`

Reply:
Done.